### ISSUE TITLE
Support for Opset 9

### ISSUE THREAD

**Issue by kkk669** [onnx/keras-onnx#30]
For example, `BatchNormalization` should not have a `spatial` attribute in opset 9. For more examples of implementation, please see https://github.com/onnx/onnxmltools/pull/241.

**Comment by vinitra** [comment]
Addressed in #31. Thanks @kkk669!
